### PR TITLE
Fix small typo in pooling_ops_test

### PR DESCRIPTION
**Pull request by ry**
(not related to pull req #1868)

### CODE REVIEW

**Comment by gunan** [comment]
Can one of the admins verify this patch?

File: tensorflow/python/kernel_tests/pooling_ops_test.py [hunk 1]
```diff
@@ -843,7 +843,7 @@ def _testAvgPoolGradSamePadding3_1(self, data_format, use_gpu):
   def testShapeFunctionEdgeCases(self):
     # All shapes unknown.
     for pool_func in [tf.nn.max_pool, tf.nn.avg_pool]:
-      p = tf.nn.max_pool(tf.placeholder(tf.float32),
+      p = pool_func(tf.placeholder(tf.float32),
                          ksize=[1, 1, 1, 1], strides=[1, 1, 1, 1],
```
Comment (vrv) [inline tensorflow/python/kernel_tests/pooling_ops_test.py]: I know it's annoying, but can you align these parameters?  Thanks for catching this.

**Comment by ry** [comment]
yep, updated

**Comment by vrv** [comment]
Thanks!  @tensorflow-jenkins test this please

**Comment by vrv** [comment]
somehow we killed jenkins, going to wait for it to restart to try again.

**Comment by vrv** [comment]
test this please

**Pull request merged**